modules/eks/ec2.tf

```diff
@@ -61,7 +61,7 @@ resource "aws_security_group_rule" "ssh_from_dmz_to_nodes" {
   from_port         = 22
   to_port           = 22
   protocol          = "tcp"
-  cidr_blocks       = local.parsed_dmz_cidr_range
+  cidr_blocks       = local.cluster_endpoint_public_access_cidrs
```

Comment:
shouldn't this be `cluster_endpoint_private_access_cidrs`?

as per:
https://github.com/sighupio/fury-eks-installer/blob/d18d4e1a5d3c939c436f03037ca9824daf404f1b/modules/eks/variables.tf#L33

Reply:
I will fix it